src/helpers/game_utils.ts

```diff
@@ -240,18 +245,18 @@ export async function getMultipleChoiceOptions(
         guessMode === GuessModeType.SONG_NAME ||
         guessMode === GuessModeType.BOTH
     ) {
+        const songName = useHangul ? "hangul_song_name" : "clean_song_name";
         easyNames = (
             await dbContext
                 .kmq("available_songs")
-                .select("clean_song_name")
-                .groupByRaw("UPPER(clean_song_name)")
+                .select(songName)
```

Comment:
Do all songs have a hangul_song_name?

Reply:
Many songs don't have a hangul_song_name

Reply:
Won't this end up with a lot of empty strings in multiple choice + korean locale

Reply:
I can add a filter for checking they're non-empty. Then, they should be populated with easy choices below